### ISSUE TITLE
First page scrolling on low resolutions

### ISSUE THREAD

**Issue by rkantos** [jfoclpf/autocosts#61]
Why does the first page of the form have to always scroll up? It is impossible to click the next button without either going full screen or zooming out the page with 1366x768 resolution.

**Comment by jfoclpf** [comment]
It scrolls to try to focus the user's attention to the form

You can undo this by commenting the content of function `scrollPage` in `documentFunctions.js`, [here](https://github.com/jfoclpf/autocosts/blob/master/src/client/documentFunctions.js#L398).

```
function scrollPage(callback){
/*
...
*/
}
```

I don't understand the "fullscreen" issue, but all the function which deal with the page as a whole are in `src/client/documentFunctions.js`

**Comment by jfoclpf** [comment]
@rkantos 
is your issue solved?

**Comment by rkantos** [comment]
> @rkantos
> is your issue solved?

Sure, originally I just thought it shouldn’t scroll on the https://autocosts.info/ -website! Another way to avoid it on lower resolutions could be to disable it if the resolution is lower, or change the layout completely.

Thanks

**Comment by jfoclpf** [comment]
Thanks @rkantos 
Just a curiosity, are you using this code on any particular project? As you know this is freeware, so you may use it freely. Just a curiosity!

**Comment by rkantos** [comment]
> Thanks @rkantos
> Just a curiosity, are you using this code on any particular project? As you know this is freeware, so you may use it freely. Just a curiosity!

No, I just used the website, and thought this would be the apporiate support channel for such an issue :) 

Mainly I am just using it to compare car costs between gasoline and electric cars. I’ve been trying to find a calculator that would have consumption numbers of cars for easy comparison.. There have been a couple, but they’ve since disappeared. :/
